src/pages/DetailPage/Components/ProductInfo/ProductInfo.scss

```
@@ -384,18 +384,84 @@
               }
             }
           }
-
-          .chosenProduct {
+          .chosenProductContainer {
+            width: 100%;
             margin-top: 30px;
-            padding: 18px 130px 20px 20px;
-            border-top: 1px solid #ddd;
-            border-bottom: 1px solid #ddd;
-            background-color: #f5f5f5;
-            color: #000;
-            font-size: 14px;
-            font-weight: 300;
-            display: flex;
-            justify-content: space-between;
+
+            .chosenProduct {
+              padding: 18px 20px 20px 20px;
```

아래와 같이 작성해주실 수 있습니다 :) 
```suggestion
              padding: 18px 20px 20px;
```

---

넵!